README.md

```diff
@@ -1,184 +1,206 @@
 # O2R-Badger
 
-API for retrieving scalable badges on reproducibility. 
+API for retrieving scalable badges for scientific publications. Used by the [o2r-extender](https://github.com/o2r-project/o2r-extender), which is a Chrome extension that integrates these badges into several research aggregators.
 
-Based on the following microservices (part of the "Badges for computational geoscience containers" study project at ifgi):
+Based on the [Badges for computational geoscience containers](https://zivgitlab.uni-muenster.de/geocontainer-badges) study project at ifgi.
 
-- [geocontainer-badges/scalability](https://zivgitlab.uni-muenster.de/geocontainer-badges/scalability)
-- [geocontainer-badges/spatial-information](https://zivgitlab.uni-muenster.de/geocontainer-badges/spatial-information)
-- [geocontainer-badges/executable-code](https://zivgitlab.uni-muenster.de/geocontainer-badges/executable-code)
-- [geocontainer-badges/licencing](https://zivgitlab.uni-muenster.de/geocontainer-badges/licencing)
-- [geocontainer-badges/release](https://zivgitlab.uni-muenster.de/geocontainer-badges/release)
-- [geocontainer-badges/peer-review](https://zivgitlab.uni-muenster.de/geocontainer-badges/peer-review)
-- [geocontainer-badges/testservers](https://zivgitlab.uni-muenster.de/geocontainer-badges/testservers)
+## Badges:
 
-Documentation of the subprojects:
+- `executable`: Information about executability and reproducibility of a publication.
+- `licence`: The license type.
+- `spatial`: Where research took place. 
+- `releasetime`: The publication date. 
+- `peerreview`: If and by which process the publication was peer reviewed.
 
-## 1 Scalable badges (geocontainer-badges/scalability)
+Services and what they are used for:
 
-This project provides an API for retrieving a scalable badge as svg or png.
+- [o2r](https://o2r.uni-muenster.de): `executable`, `licence`, `location`
+- [crossref](https://www.crossref.org/): `releasetime`
+- [DOAJ](https://doaj.org/):  `peerreview`
 
-### Installation
+Note that the badger will only find results if the respective service has the publication listed.
 
-In order to use this API, make sure that you have installed [Docker](https://www.docker.com/) and [Node.js] (https://nodejs.org/en/).
+## Requirements
 
-Then clone our repository:
+- Node.js `>= 6.2` and npm (optional; for development)
+- docker
+- Access to the following services:
+    - o2r API (https://o2r.uni-muenster.de)
+    - Crossref API (https://www.crossref.org/)
+    - DOAJ API (https://doaj.org/)
+    - GeoNames API (http://geonames.org)
 
-`git clone git@zivgitlab.uni-muenster.de:geocontainer-badges/scalability.git`
+## Remote installation
```

Comment:
Yes, but the part now called "local installation" is also container-based. So maybe "Installation via Docker Hub" and "Manual installation"?

Or I just call it "Installation" and list the two/three options available:
- Docker Hub 
- Docker (building the image manually)
- Node.js